packages/xo-web/src/common/notification.js

```diff
@@ -58,10 +58,12 @@ export class Notification extends Component {
               ) : (
                 body
               ),
-              6e3
+              autoClose
```

Comment:
Is it possible to never hide it?

Reply:
Do you mean make `autoClose` parameter required ?